src/terrain/services/requests.clj

```diff
@@ -67,14 +67,38 @@
   "Marks a request as being in progress."
   (request-update-fn "Your request is in progress." "in-progress"))
 
-(def request-rejected
+(def mark-request-rejected
   "Marks a request as having been rejected."
   (request-update-fn "No deinal reason given." "rejected"))
```

Comment:
Looks like a tpyo here (outside of your PR changes, but figured I'd mention it):
```suggestion
  (request-update-fn "No denial reason given." "rejected"))
```

Reply:
Ah, I completely missed that. Thanks for catching it.